package.json

```diff
@@ -21,7 +22,10 @@
   "dependencies": {
     "babel": "^6.23.0",
```

Comment:
Some of these dependencies are developer dependencies. Faker and babel for example.